src/registry_schemas/example_data/schema_data.py

```diff
@@ -781,30 +780,6 @@
                 'postalCode': 'H0H0H0',
                 'addressRegion': 'BC'
             },
-            'roles': [
-                {
-                    'roleType': 'Completing Party',
```

Comment:
@thorwolpert If I understood correctly dissolution does not contain `Completing Party` , so removing it from here.

Reply:
I think every filing has a completing party.